package.json

```diff
@@ -138,6 +138,7 @@
     "prettier": "^2.0.5",
     "prettier-eslint": "^10.1.1",
     "typescript": "^4.4.4",
-    "web-vitals": "^2.1.2"
+    "web-vitals": "^2.1.2",
+    "@babel/plugin-proposal-private-property-in-object": "^7.14.5"
```

Comment:
This package has been deprecated, you should replace it by `@babel/plugin-transform-private-property-in-object`.